strictdoc/backend/excel/import_/excel_to_sdoc_converter.py

```diff
@@ -43,23 +42,22 @@ class ExcelToSDocConverter:
     @staticmethod
     # optional argument title is present for external scripts to assign a title
     def convert(excel_file, title=None) -> SDocDocument:
```

Comment:
This signature could use type hints, esp on `excel_file`. Looks like it's a `str`, coming from argparse. This need for type hints propagates all the way up to [`ImportExcelCommandConfig`](https://github.com/strictdoc-project/strictdoc/blob/main/strictdoc/cli/cli_arg_parser.py#L62). And `title` should be `Union[str, None]`.